functions.php

```diff
@@ -65,54 +74,36 @@ function setup() : void {
 function enqueue_styles() : void {
 
 	$theme_version  = wp_get_theme()->get( 'Version' );
+
 	$parent_version = wp_get_theme( 'twentytwenty' )->get( 'Version' );
 
+	// Enqueue fonts
+	wp_enqueue_style( 'mutualaidnyc-fonts', mutualaidnyc_fonts_url(), array(), null );
+
+	// Theme variables
+	wp_enqueue_style( 'mutualaid-variables-style', get_stylesheet_directory_uri() . '/assets/variables.css', array(), wp_get_theme()->get( 'Version' ) );
+
 	wp_enqueue_style( 'parent-style', get_template_directory_uri() . '/style.css', [], $parent_version );
-	wp_enqueue_style(
-		'font-poppins',
-		'https://fonts.googleapis.com/css2?family=Poppins:wght@700;900&display=swap',
-		[],
-		$theme_version
-	);
-	wp_enqueue_style(
-		'font-francois-one',
-		'https://fonts.googleapis.com/css2?family=Francois+One&display=swap',
-		[],
-		$theme_version
-	);
-	wp_enqueue_style(
-		'font-karla',
-		'https://fonts.googleapis.com/css2?family=Karla:ital,wght@0,400;0,700;1,400;1,700&display=swap',
-		[],
-		$theme_version
-	);
+
 	wp_enqueue_style( 'theme-style', get_stylesheet_uri(), [ 'parent-style' ], $theme_version );
 }
 
 /**
- * Enqueues the block editor styles.
+ * Add Google webfonts
  *
- * @return void
+ * @return string
  */
-function block_editor_styles() : void {
-	$theme_version = wp_get_theme()->get( 'Version' );
-	wp_enqueue_style(
-		'font-poppins',
-		'https://fonts.googleapis.com/css2?family=Poppins:wght@700;900&display=swap',
-		[],
-		$theme_version
-	);
-	wp_enqueue_style(
-		'font-francois-one',
-		'https://fonts.googleapis.com/css2?family=Francois+One&display=swap',
-		[],
-		$theme_version
-	);
-	wp_enqueue_style(
-		'font-karla',
-		'https://fonts.googleapis.com/css2?family=Karla:ital,wght@0,400;0,700;1,400;1,700&display=swap',
-		[],
-		$theme_version
-	);
-	wp_enqueue_style( 'theme-block-style', get_theme_file_uri( 'assets/styles/editor-styles.css' ), [], $theme_version );
-}
+function mutualaidnyc_fonts_url() : string {
+	$fonts_url = '';
+
+	$font_families = array();
+	$font_families[] = 'family=Poppins:wght@700;900';
+	$font_families[] = 'family=Francois+One';
+	$font_families[] = 'family=Karla:ital,wght@0,400;0,700;1,400;1,700';
+	$font_families[] = 'display=swap';
+
+	// Make a single request for the theme fonts
+	$fonts_url = 'https://fonts.googleapis.com/css2?' . implode( '&', $font_families );
+
+	return $fonts_url;
```

Comment:
One reason I enqueued the font families multiple times was I found the additional `family=` query var was being removed by WP's URL sanitization, so only one was left. Have you seen that?

Reply:
I did encounter that! Couldn't figure out how to get this to work with `add_query_arg()`, so I went with a different approach.

Reply:
Oh interesting, this works? Interesting. Happy to merge if it looks good on your end 👍

Reply:
Yep, just tested and it's okay. 🚢